ci/infra/libvirt/README.md

```diff
@@ -35,6 +35,8 @@ Copy the `terraform.tfvars.example` to `terraform.tfvars` and provide reasonable
 
 ## Variables
 
+`libvirt_uri` - URL of the libvirt host
```

Comment:
I think we should add also a section in the testrunner's `README.md` for this libvirt specific variables.

Reply:
Yes doing it right now :+1: